python/dgl/nn/pytorch/conv/edgegatconv.py

```diff
@@ -0,0 +1,381 @@
+"""Torch modules for graph attention networks(GAT)."""
+# pylint: disable= no-member, arguments-differ, invalid-name
+import torch as th
+from torch import nn
+
+from .... import function as fn
+from ....base import DGLError
+from ....utils import expand_as_pair
+from ...functional import edge_softmax
+
+# pylint: enable=W0235
+class EdgeGATConv(nn.Module):
```

Comment:
Maybe mention SCENE in the name of EdgeGATConv? Since we already have another similar egatconv https://github.com/dmlc/dgl/blob/master/python/dgl/nn/pytorch/conv/egatconv.py